homeassistant/components/bluecurrent/const.py

```diff
@@ -0,0 +1,190 @@
+"""Constants for the BlueCurrent integration."""
+import logging
+
+from homeassistant.components.sensor import SensorEntityDescription
+
+DOMAIN = "bluecurrent"
+
+LOGGER = logging.getLogger(__package__)
+
+PLATFORMS = ["sensor"]
+CHARGE_POINTS = "CHARGE_POINTS"
+DATA = "data"
+DELAY_1 = 1
+DELAY_2 = 20
+
+EVSE_ID = "evse_id"
+GRID_STATUS = "GRID_STATUS"
+MODEL_TYPE = "model_type"
+OBJECT = "object"
+TIMESTAMP_KEYS = ("start_datetime", "stop_datetime", "offline_since")
+VALUE_TYPES = "CH_STATUS"
```

Comment:
Same here – if these are only related to the API, I'm not sure they need to be shared consts.

Reply:
Moved them to `__init__.py`.